frontend/src/individual_certificate.tsx

```diff
@@ -446,15 +446,15 @@ export function IndividualCertificate({
           )}
           <div className={style.paneControls}>
             <a
-              target="_blank"
```

Comment:
Why did this disappear?

Reply:
Not sure if we need to both delete it and add `rel` (but don't think it hurts).

Since this should be a download, why should there be a new window?